### ISSUE TITLE
Dev env needs fixtures for working submit form

### ISSUE THREAD

**Issue by conorsch** [freedomofpress/pressfreedomtracker.us#563]
The RECAPTCHA changes pulled in via #524 work fine in prod, but break the local dev environment due to lack of a working site key. The form does not render:

![tracker-submit-form-does-not-draw](https://user-images.githubusercontent.com/657862/33580905-b9c45ac0-d903-11e7-910d-9ea928d5ebbe.png)

A peek at the console shows that the site-key parameter is missing:

![tracker-submit-form-console-error](https://user-images.githubusercontent.com/657862/33580918-c2c6b168-d903-11e7-958b-49c5e2a216b8.png)

Relevant docs for upstream projects:

* https://github.com/springload/wagtail-django-recaptcha
* https://github.com/praekelt/django-recaptcha#local-development-and-functional-testing

Likely we can futz with the `settings/*.py` files and restore functionality for local development.

**Comment by harrislapiroff** [comment]
This is a pretty old issue and I think not relevant anymore.
